typescript/solver/solvers/eco/config/metadata.ts

```diff
@@ -0,0 +1,19 @@
+import type { EcoMetadata } from "../types.ts";
+
+const metadata: EcoMetadata = {
+  protocolName: "Eco",
+  intentSource: {
+    address: "0x734a3d5a8D691d9b9,11674E682De5f06517c79ec",
+    chainId: 11155420,
+    chainName: "optimism-sepolia",
```

Comment:
Re: the above comment, imo we should be consistent to using chainNames or domain numbers. IMO, I would standardize on chainnames, and also enforce the names from the registry to avoid custom translation